### PR TITLE
2023-11-01 swagger

### PR DESCRIPTION
**Pull request by xiangyan99**
This is the stable version for ignite.

### CODE REVIEW

File: sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py [hunk 1]
```diff
@@ -473,140 +472,3 @@ def get_service_statistics(self, **kwargs: Any) -> MutableMapping[str, Any]:
         kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
-
-    @distributed_trace
-    def list_aliases(self, *, select: Optional[List[str]] = None, **kwargs: Any) -> ItemPaged[SearchAlias]:
```
Comment (pvaneck) [inline sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py:478]: The removal of all these alias methods seem to be missing from the CHANGELOG.

File: sdk/search/azure-search-documents/azure/search/documents/indexes/models/__init__.py [hunk 2]
```diff
@@ -34,16 +34,15 @@
...
 from .._generated.models import (
-    SearchAlias,
-    AzureMachineLearningSkill,
     AnalyzeResult,
     AnalyzedTokenInfo,
     AsciiFoldingTokenFilter,
-    AzureOpenAIEmbeddingSkill,
-    AzureOpenAIParameters,
-    AzureOpenAIVectorizer,
+    BlobIndexerDataToExtract,
```
Comment (pvaneck) [inline sdk/search/azure-search-documents/azure/search/documents/indexes/models/__init__.py:40]: There seem to be a lot of new model classes exposed in the public API. Should any of these be mentioned in the CHANGELOG?

File: sdk/search/azure-search-documents/azure/search/documents/_search_client.py [hunk 3]
```diff
@@ -137,7 +134,7 @@ def get_document(self, key: str, selected_fields: Optional[List[str]] = None, **
     @distributed_trace
     def search(
         self,
-        search_text: str,
+        search_text: Optional[str] = "",
```
[the remark this answers is not in the document]
Comment (xiangyan99) [reply sdk/search/azure-search-documents/azure/search/documents/_search_client.py]: Good point. Let's use `None`.

File: sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py (hunk 1 above)
Comment (xiangyan99) [reply sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py:478]: Added. :)

File: sdk/search/azure-search-documents/azure/search/documents/indexes/models/__init__.py (hunk 2 above)
Comment (xiangyan99) [reply sdk/search/azure-search-documents/azure/search/documents/indexes/models/__init__.py:40]: Added.

File: sdk/search/azure-search-documents/azure/search/documents/_index_documents_batch.py [hunk 4]
```diff
@@ -40,9 +40,9 @@ def add_upload_actions(self, *documents: Union[List[Dict], List[List[Dict]]]) ->
 
         :param documents: Documents to upload to an Azure search index. May be
          a single list of documents, or documents as individual parameters.
-        :type documents: Dict or List[Dict]
+        :type documents: dict or list[Dict]
```
[the remark this answers is not in the document]
Comment (xiangyan99) [reply sdk/search/azure-search-documents/azure/search/documents/_index_documents_batch.py]: Updated

**Review by pvaneck — APPROVED** [review]

**Pull request merged**